include/fg/font.h

```diff
@@ -10,10 +10,24 @@
 #pragma once
 #include <fg/defines.h>
 
-namespace internal
-{
-class _Font;
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+FGAPI fg_err fg_create_font(fg_font* pFont);
```

Comment:
C-API doesn't have any documentation. I will do another round for that once the functionality is working.

It doesn't load any default font with just create call, again that will be clear once i update it with documentation. `pFont` is only used if `fg_create_font` is called upon, otherwise it is an irrelevant variable.